pfcpiface/utils.go

```diff
@@ -99,3 +99,14 @@ func maxUint64(x, y uint64) uint64 {
 func calcBurstSizeFromRate(kbps uint64, ms uint64) uint64 {
 	return uint64((float64(kbps) * 1000 / 8) * (float64(ms) / 1000))
 }
+
+func getRemoteIP(dstIP string) (net.IP, error) {
```

Comment:
I'm not comfortable with this function due to:
- `PFCPPort` is hardocoded.
- the function does two things: establishes connection to a peer, and then retrieves IP address from the established connection. 

I propose to get rid of this function, and move this piece of code to `tryConnectToN4Peer()` (see my comment above).

Reply:
Done